docs/getting-started/tutorial-sample-data.asciidoc

```diff
@@ -1,206 +1,160 @@
-. In the *Average Ticket Price* visualization, click the gear icon in 
-the upper right.
-. From the *Options* menu, select *Edit visualization*.
-+
-*Average Ticket Price* is a metric visualization.
-To specify which groups to display 
-in this visualization, you use an {es} {ref}/search-aggregations.html[bucket aggregation]. 
-This aggregation sorts the documents that match your search criteria into different 
-categories, or buckets.
+[[filter-and-query-the-data]]
+==== Filter and query the data
 
-[float]
-==== Create a bucket aggregation
+To focus in on the data you want to explore, use filters and queries. 
+For more information, refer to 
+{ref}/query-filter-context.html[Query and filter context].
 
-. In the *Buckets* pane, select *Add > Split group*.
-. In the *Aggregation* dropdown, select *Terms*.
-. In the *Field* dropdown, select *Carrier*.
-. Set *Descending* to *4*.
-. Click *Apply changes* image:images/apply-changes-button.png[].
+To filter the data:
+
+. In the *Controls* visualization, select an *Origin City* and *Destination City*, then click *Apply changes*.
 +
-You now see the average ticket price for all four airlines.
+The `OriginCityName` and the `DestCityName` fields filter the data in the panels. 
 +
-[role="screenshot"]
-image::getting-started/images/tutorial-sample-edit1.png[]
-
-[float]
-==== Save the visualization
-
-. In the menu bar, click *Save*.
-. Leave the visualization name as is and confirm the save.
-. Go to the *Global Flight* dashboard and scroll the *Average Ticket Price* visualization to see the four prices.
-. Optionally, edit the dashboard. Resize the panel 
-for the *Average Ticket Price* visualization by dragging the
-handle in the lower right. You can also rearrange the visualizations by clicking 
-the header and dragging. Be sure to save the dashboard.
+For example, the following dashboard shows the data for flights from London to Milan. 
 +
 [role="screenshot"]
-image::getting-started/images/tutorial-sample-edit2.png[]
+image::getting-started/images/tutorial-sample-filter.png[]
 
-[float]
-[[tutorial-sample-inspect]]
-=== Inspect the data
+. To manually add a filter, click *Add filter*,
+then specify the data you want to view.
+
+. When you are finished experimenting, remove all filters.
 
-Seeing visualizations of your data is great, 
-but sometimes you need to look at the actual data to
-understand what's really going on. You can inspect the data behind any visualization
-and view the {es} query used to retrieve it.
+[[query-the-data]]
+To query the data:
 
-. In the dashboard, hover the pointer over the pie chart, and then click the icon in the upper right. 
-. From the *Options* menu, select *Inspect*.
+. To view all flights out of Rome, enter the following in the *KQL* query bar, then click *Update*:
++
+[source,text]
+OriginCityName:Rome   
+
+. For a more complex query with AND and OR, enter:
++
+[source,text]
+OriginCityName:Rome AND (Carrier:JetBeats OR "Kibana Airlines")
```

Comment:
I think this query is being interpreted as any documents that have the text "Kibana Airlines" in any field, it would be more explicit to say:

```suggestion
OriginCityName:Rome AND (Carrier:JetBeats OR Carrier:"Kibana Airlines")
```